src/plugin/cesium/tiles/cesium3dtileslayer.js

```diff
@@ -65,10 +77,24 @@ plugin.cesium.tiles.Layer.prototype.synchronize = function() {
 
   if (!this.hasError()) {
     var tilesetUrl = '';
-    if (!isNaN(this.assetId)) {
+    if (!isNaN(this.assetId) && this.accessToken) {
       tilesetUrl = Cesium.IonResource.fromAssetId(this.assetId, {
         accessToken: this.accessToken
       });
+
+      tilesetUrl.then(() => {
+        // We don't care to note that it resolves, we just want a response if it doesn't
+      }, () => {
+        // Clear the saved access token because it was rejected
+        const layerId = this.getId();
+        os.settings.set(`plugin.cesium.SettingsKey.ACCESS_TOKEN.${layerId}`, '');
```

Comment:
I initially thought it might be nice to save an access token per layer, but I'm back tracking on that. The default token in Ion has read for all assets, and you can only select individual assets by creating a new token. A user would be most likely to use their default token (and we instruct them to do so), so I don't think we need to save multiple.

Side note, the template string here needs an expression (`${...}`) around `plugin.cesium.SettingsKey.ACCESS_TOKEN`. It's currently being used as a literal setting key instead of evaluating the property to get the correct key. With the above suggestion you'll just use `os.settings.set(plugin.cesium.SettingsKey.ACCESS_TOKEN, <value>)` though.